application/clicommands/SendCommand.php

```diff
@@ -42,6 +44,8 @@ class SendCommand extends Command
      *   --no-acknowledge           Do not acknowledge Icinga problem
      *   --command-pipe <path>      Legacy command pipe, allows to run without
      *                              depending on a configured monitoring module
+     *   --icingadb                 Use icingadb as backend, default to "monitoring" if
```

Comment:
*default* should be *defaults*. `"monitoring"` (in quotes) implies this could be another switch or a value for it. I'd write this instead: `..., the IDO backend (if available) is used if not passed`